src/main/java/com/baidu/hugegraph/perf/NormalStopwatch.java

```diff
@@ -0,0 +1,306 @@
+        }
+        if (this.maxCost < cost) {
+            this.maxCost = cost;
+        }
+
+        this.totalCost += cost;
+        this.totalSelfWasted += wastedTime;
+        this.lastStartTime = -1L;
+    }
+
+    @Override
+    public long times() {
+        return this.times;
+    }
+
+    @Override
+    public long totalCost() {
+        return this.totalCost;
+    }
+
+    @Override
+    public void totalCost(long totalCost) {
+        this.totalCost = totalCost;
+    }
+
+    @Override
+    public long minCost() {
+        return this.minCost;
+    }
+
+    @Override
+    public long maxCost() {
+        return this.maxCost;
+    }
+
+    @Override
+    public long totalTimes() {
+        if (this.totalChildrenTimes > 0L) {
+            return this.times + this.totalChildrenTimes;
+        }
+        return this.times;
+    }
+
+    @Override
+    public long totalChildrenTimes() {
+        return this.totalChildrenTimes;
+    }
+
+    @Override
+    public long totalWasted() {
+        if (this.totalChildrenWasted > 0L) {
+            return this.totalSelfWasted + this.totalChildrenWasted;
+        }
+        return this.totalSelfWasted;
+    }
+
+    @Override
+    public long totalSelfWasted() {
+        return this.totalSelfWasted;
+    }
+
+    @Override
+    public long totalChildrenWasted() {
+        return this.totalChildrenWasted;
+    }
+
+    @Override
+    public void fillChildrenTotal(List<Stopwatch> children) {
+        // Fill total wasted cost of children
+        this.totalChildrenWasted = children.stream().mapToLong(
+                                   c -> c.totalWasted()).sum();
+        // Fill total times of children
+        this.totalChildrenTimes = children.stream().mapToLong(
+                                  c -> c.totalTimes()).sum();
+    }
+
+    @Override
+    public Stopwatch copy() {
+        try {
+            return (Stopwatch) super.clone();
+        } catch (CloneNotSupportedException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public Stopwatch child(String name) {
+        return this.children.get(name);
+    }
+
+    @Override
+    public Stopwatch child(String name, Stopwatch watch) {
+        if (watch == null) {
+            return this.children.remove(name);
+        }
+        return this.children.put(name, watch);
+    }
+
+    @Override
+    public boolean empty() {
+        return this.children.size() == 0;
+    }
+
+    @Override
+    public void clear() {
+        this.lastStartTime = -1L;
+
+        this.times = 0L;
+        this.totalCost = 0L;
+
+        this.minCost = Long.MAX_VALUE;
+        this.maxCost = 0L;
+        this.totalSelfWasted = 0L;
+        this.totalChildrenWasted = -1L;
+        this.totalChildrenTimes = -1L;
+
+        this.children.clear();
+    }
+
+    @Override
+    public String toString() {
+        return String.format("{parent:%s,name:%s," +
+                             "times:%s,totalChildrenTimes:%s" +
```

Comment:
add',' at the end of the line, and sometimes there are blank space after',', sometimes there aren't.